docs/source/examples/cirq-ibmq-backends.md

```diff
@@ -70,7 +69,7 @@ the expectation value to mitigate. This function will:
 2. Run the circuit.
 3. Convert from raw measurement statistics (or a different output format) to an expectation value.
 
-For information on how to define more advanced executors, see the {ref}`Executors <guide/executors/executors>` section of the Mitiq user guide.
+For information on how to define more advanced executors, see the {doc}`../guide/executors` section of the Mitiq user guide.
```

Comment:
Great question! I had to figure that out too.

- The [`{ref}`](https://www.sphinx-doc.org/en/master/usage/referencing.html#role-ref) role is an all-purpose link-creating-tool for creating references to documents **and** labeled sections of documents (e.g. an anchor within a specific page).
- The [`{doc}`](https://www.sphinx-doc.org/en/master/usage/referencing.html#role-doc) role is specifically made for referencing **local** documents only (no external documents, and no anchors/sections/equations/images/etc). One thing to note here with `{doc}`, is that if we do not pass text in for the link, it uses the title of the page we are linking to.
- It's worth mentioning that both of these can be achieved with markdown links `[text](link)` as well, so why bother with either of these roles? When using `{ref}` and `{doc}`, we are providing sphinx with more information than it would get otherwise, and is able to some magic and make our lives easier. I'm not totally clear on what, but I think it makes refactoring easier as sphinx will know that specific references are broken using this syntax.

As for why sphinx has both `{doc}` and `{ref}`, I'm not sure. It does seem a bit redundant to me. Why not just have the more powerful one? That said, I always like the idea of using the simplest solution to a problem, and that includes using the simplest available tool.